### ISSUE TITLE
SetFullScreen(false) give error

### ISSUE THREAD

**Issue by AnkitKumar2698** [fyne-io/fyne#2588]
When I am exiting from full screen I got the exit status 2  The error is given below in Bold
**panic: InvalidValue: Invalid window size 0x0

goroutine 1 [running, locked to thread]:
github.com/go-gl/glfw/v3.3/glfw.acceptError({0x0, 0x0, 0x3c})
        C:/Users/Ankit Sharma/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20210410170116-ea3d685f79fb/error.go:174 +0x186
github.com/go-gl/glfw/v3.3/glfw.panicError(...)
        C:/Users/Ankit Sharma/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20210410170116-ea3d685f79fb/error.go:185
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetMonitor(0xc0000b2000, 0xc000032088, 0xc000032148, 0x1, 0x0, 0x7ff752bbd54b, 0x0)
        C:/Users/Ankit Sharma/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20210410170116-ea3d685f79fb/window.go:692 +0x35
fyne.io/fyne/v2/internal/driver/glfw.(*window).SetFullScreen.func1()
        C:/Users/Ankit Sharma/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/window.go:136 +0x9b
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).runGL(0xc00007e910)
        C:/Users/Ankit Sharma/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/loop.go:104 +0x197
        C:/Users/Ankit Sharma/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/driver.go:83 +0x30
fyne.io/fyne/v2/internal/driver/glfw.(*window).ShowAndRun(0xc0000b2000)
        C:/Users/Ankit Sharma/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/window.go:462 +0x2f
main.main()
        C:/Users/Ankit Sharma/Desktop/golang crazy/project/os.go:33 +0x157
exit status 2**

### To Reproduce:
Steps to reproduce the behaviour:
1.  create a button for SetFullScreen false  
2. Click on it after building
3. See error in cmd and GUI window will be exited

### Example code:
```go
package main

import (
	// "fmt"
	// "fyne.io/fyne/v2"
	"fmt"

	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/widget"
)

func main(){
	a:=app.New()
	w:=a.NewWindow("ALinux")
	fullScreen:=true;
	fullScreenBtn:=widget.NewButton("Full Screen",func(){
		w.SetFullScreen(fullScreen)
		fullScreen=!fullScreen
	})
	
	w.SetContent(fullScreenBtn)
	w.ShowAndRun()
}
```

### Device (please complete the following information):
Windows 10
go version go1.17.1 windows/amd64
fyne cli version: v2.1.1

**Comment by MatejMagat305** [comment]
so I was debuging and I do not know where is bug (my modification work, but I was looked at content)....., I send screenshot after second pressing above code....
![2021-10-29_17-03](https://user-images.githubusercontent.com/61238240/139484997-4f3070dc-c21e-4616-807d-4a1ef2ce6e05.png)

**Comment by andydotxyz** [comment]
Fixed on `release/v2.1.x` for v2.1.2
